crates/rspack_plugin_schemes/src/file_uri.rs

```diff
@@ -37,6 +42,30 @@ async fn normal_module_factory_resolve_for_scheme(
   Ok(None)
 }
 
+#[plugin_hook(NormalModuleReadResource for FileUriPlugin,tracing=false)]
+async fn read_resource(
+  &self,
+  resource_data: &ResourceData,
+  fs: &Arc<dyn ReadableFileSystem>,
+) -> Result<Option<Content>> {
+  let scheme = resource_data.get_scheme();
+  if scheme.is_none()
```

Comment:
This is moved from `crates/rspack_loader_runner/src/runner.rs`, which is the same as webpack: https://github.com/webpack/webpack/blob/9f98d803c0c5011188901e895a7060ab06e12f80/lib/schemes/FileUriPlugin.js#L39-L48